hybris/platforms/common/server_wlegl.cpp

```diff
@@ -62,10 +56,7 @@ server_wlegl_create_handle(struct wl_client *client,
 		return;
 	}
 
-	handle = server_wlegl_handle_create(id);
-	wl_array_copy(&handle->ints, ints);
-	handle->num_fds = num_fds;
-	wl_client_add_resource(client, &handle->resource);
```

Comment:
Was this completely unnecessary or does one of the new calls in server_wlegl_handle_create do what this did?

Reply:
has been moved to `server_wlegl_handle_create` in order hide implementation details and simplify the calling code

Reply:
I checked the wayland code and it does indeed wl_resource_create does also do internally the same things as wl_client_add_resource.

Reply:
that's why I was calling commit "wayland: Stop using wl_resource internals in server_wlegl_handle" ;p

a forthcoming patch would move to wayland-*-core and hide protocol implementation details, thus making hybris more ABI safe